libp2p/protocols/rendezvous.nim

```diff
@@ -511,6 +508,11 @@ proc advertisePeer(rdv: RendezVous, peer: PeerId, msg: seq[byte]) {.async.} =
 proc advertise*(
     rdv: RendezVous, ns: string, ttl: Duration, peers: seq[PeerId]
 ) {.async.} =
+  ## The advertise async procedure sends advertisements for a namespace
```

Comment:
What advertisements are?

Reply:
I change advertisements to registration, as it makes more sense.